### PR TITLE
[Behat] Checkout adressing initial step adjustment

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by lchrusciel** [comment]
Build is [green](https://travis-ci.org/github/Sylius/Sylius/builds/708371864) 💚

File: features/checkout/addressing_order/having_default_address_preselected.feature [hunk 1]
```diff
@@ -15,7 +15,7 @@ Feature: Having a default address preselected
     @ui
     Scenario: Having a default address preselected on checkout addressing step
         When I add product "PHP T-Shirt" to the cart
-        And I am at the checkout addressing step
+        And I am addressing my order
```
Comment (GSadee) [inline features/checkout/addressing_order/having_default_address_preselected.feature:18]: IMO unnecessary step in this scenario

File: features/checkout/addressing_order/having_only_possible_country_preselected.feature [hunk 2]
```diff
@@ -13,12 +13,12 @@ Feature: Having only possible country preselected
     @ui
     Scenario: Having the only country preselected on addressing form
         When I add product "PHP T-Shirt" to the cart
-        And I am at the checkout addressing step
+        And I am addressing my order
```
Comment (GSadee) [inline features/checkout/addressing_order/having_only_possible_country_preselected.feature:16]: Similar to the above

File: features/checkout/paying_for_order/sorting_payment_method_selection.feature [hunk 3]
```diff
@@ -16,7 +16,7 @@ Feature: Sorting payment method selection
     @ui
     Scenario: Seeing payment methods sorted
         Given I have product "Targaryen T-Shirt" in the cart
-        When I am at the checkout addressing step
+        When I am addressing my order
```
Comment (GSadee) [inline features/checkout/paying_for_order/sorting_payment_method_selection.feature:19]:
```suggestion
        And I am addressing my order
```
here and in some other scenarios

**Pull request closed without merging**